## Урок 5.

### Применение математической статистики для проверки гипотез в реальной жизни для популярных метрик

#### 1) Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде .Результаты: 1)Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 3) Конверсии : conv1 = 1.05% conv2 = 1.47% .

Для оценки статистической значимости между конверсиями при a = 5% можно использовать Z критерий. Формула Z критерия:
Z = (p1-p2)/(p(1-p)*(1/N1 + 1/N2)**0.5,        p = (n1 * p1 + n2 * p2) / (n1 + n2)

p1, p2 - конверсия в группах, N1, N2 - количество пользователей в группах, которые заходили на сайт во время эксперимента, n1, n2  - количество прользователей в группах, сделавших хотябы одну покупку

In [3]:
N1 = 15550
N2 = 15550
n1 = 162
n2 = 228
p1 = 164 / 15550
p2 = 228 / 15550

In [12]:
p =  (n1 * p1 + n2 * p2) / (n1 + n2)
Z = round(((p1 - p2) /((p * (1 - p) * (1/N1 + 1/N2))**0.5)),4)
Z

-3.2096

Критическое значение  Z критерия  при a = 5% и двухсторонней альтернативе равно 1.96. По причине толь, что расчётное значение Z меньше: -3.2096 < 1.96, отвергаем нулевую гипотезу о равенстве конверсий в группах и, следовательно, сделанные изменения принесли статистически значимый результат. Рекомендуем использовать новую форму заказа.

#### 2) Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

Если размер выборки достаточно большой (более 30ти элементов в группе), может быть использован Z критерий. Так как размер выборки 10000 в группе, то будем использовать Z критерий.